.github/workflows/run-benchmark.yml

```diff
@@ -29,6 +29,13 @@ jobs:
       - name: Configure git
         run: git config --global --add safe.directory "${GITHUB_WORKSPACE}"
 
+      - name: Get Penny token
```

Comment:
Why so high though, I'd move it to right before the comment acrion.

Reply:
As you wish master